release_notes.md

```diff
@@ -1,18 +1,55 @@
 # What's New
 
-### Advanced Filtering for Enhanced Precision  
+## Connector Generation
 
-The **Advanced Filtering** feature enhances FortiSOAR's search capabilities by allowing users to apply specific criteria to filter records. Whether it's filtering by severity, status, or priority, this functionality ensures that users can retrieve exactly the data they need, improving operational efficiency and decision-making.
+FortiAI now enables connector creation using natural language. By providing inputs such as CURL commands, API details, and configuration parameters, users can generate a fully functional connector that includes Connector configuration, Auto-generated connector code, and Sample playbooks.
 
-### Voice Recognition for Hands-Free Interaction  
+You can also upload attachments to assist in connector generation.
 
-The **Voice Recognition** feature transforms the way users interact with FortiSOAR by enabling hands-free control. This innovation streamlines SOC tasks, offering users greater flexibility to issue commands and retrieve data efficiently using natural language.  
+## Advanced Playbook Generation
 
-> [!Note]
-> The *Voice Recognition* feature is currently unsupported on the Firefox browser as the webkit `SpeechRecognition` is not compatible with Mozilla Firefox. Hence, the mic button is not available when the FortiSOAR&trade; environment is accessed using the Firefox browser.
+FortiAI now produces more accurate, context-aware playbook blocks tailored to specific use cases. Whether automating standard workflows or designing complex incident response logic, it offers an efficient starting point that reduces manual effort.
+
+Key improvements include:
+
+- Playbook creation from simple prompts or clicks
+- Accurate detection of connector step types, operation names, and available inputs
+- Identification of steps that require loops for record traversal
+
+### Improved Jinja Expression Selection
+
+FortiAI intelligently selects appropriate Jinja expressions for each playbook step, reducing the need for manual adjustments.
+
+Structured outputs are now supported, with JSON data displayed in a readable JSON editor.
+
+## Configuration Wizard Enhancements
 
-**Conversation Thread Management**
+The configuration wizard has been improved to support the following:
 
-Previously, conversation threads were deleted from OpenAI whenever *Clear Conversation* was clicked, and a new thread was started with each new prompt.
+- Creation of SOC and Playbook assistants
```

Comment:
Creation of SOC and Playbook assistants --> this was already present. Connetor assistant is new this time